lib/src/main/java/com/smileidentity/util/Util.kt

```diff
@@ -32,16 +32,59 @@ import androidx.core.graphics.scale
 import androidx.core.text.getSpans
 import androidx.exifinterface.media.ExifInterface
 import com.google.mlkit.vision.common.InputImage
+import com.smileidentity.R
 import com.smileidentity.SmileID
 import com.smileidentity.SmileID.moshi
 import com.smileidentity.SmileIDCrashReporting
+import com.smileidentity.compose.consent.bvn.BvnOtpVerificationMode
 import com.smileidentity.models.SmileIDException
 import kotlinx.coroutines.CoroutineExceptionHandler
 import retrofit2.HttpException
 import timber.log.Timber
 import java.io.File
 import java.io.Serializable
 
+internal fun createBvnOtpVerificationModes(maps: List<Map<String, String>>):
```

Comment:
Use the `typealias` you created?
```suggestion
internal fun createBvnOtpVerificationModes(maps: List<BvnVerificationMode>):
```